File: Examples/VisualBasic/ManageBarcodeImages/BarcodeImageQuality.vb
Imports System.IO
Imports Aspose.BarCode
Imports Aspose.BarCode.Generation

Namespace Aspose.BarCode.Examples.VisualBasic.ManageBarCodeImages
    Class BarcodeImageQuality
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_ManageBarCodesImages()

            ' Instantiate barcode object and set CodeText, Barcode Symbology and image quality mode to anti alias
            Dim barCodeBuilder As New BarCodeBuilder("1234567", EncodeTypes.Code128) With { _
                .ImageQuality = ImageQualityMode.AntiAlias _
            }
            BarCodeBuilder.Save(dataDir & Convert.ToString("barcode-image-quality_out.jpeg"), System.Drawing.Imaging.ImageFormat.Jpeg)
        End Sub
    End Class
End Namespace

File: Examples/CSharp/ManageBarcodeImages/BarcodeImageQuality.cs
using System.IO;
using Aspose.BarCode;
using System;
using Aspose.BarCode.Generation;

namespace Aspose.BarCode.Examples.CSharp.ManageBarCodeImages
{
    class BarcodeImageQuality
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_ManageBarCodesImages();

            // Instantiate barcode object and set CodeText, Barcode Symbology and image quality mode to anti alias
            BarCodeBuilder barCodeBuilder = new BarCodeBuilder("1234567", EncodeTypes.Code128)
            {
                ImageQuality = ImageQualityMode.AntiAlias
            };
            barCodeBuilder.Save(dataDir + "barcode-image-quality_out.jpeg", System.Drawing.Imaging.ImageFormat.Jpeg);
        }
    }
}
